src/locales/vocabulary.js

```diff
@@ -54,23 +54,24 @@ export default {
   'featurepanel.error_network': "Can't get the feature, check your network cable.",
   'featurepanel.error_deleted': 'This object is marked as deleted in OpenStreetMap.',
   'featurepanel.history_button': 'History »',
-  'featurepanel.other_info_heading': 'More information',
+  'featurepanel.other_info_heading': 'Details',
   'featurepanel.edit_button_title': 'Edit in OpenStreetMap database',
-  'featurepanel.edit_button': 'Edit place',
-  'featurepanel.add_place_button': 'Add place',
+  'featurepanel.note_button': 'Suggest an edit',
+  'featurepanel.edit_button': 'Edit details',
+  'featurepanel.add_place_button': 'Add a place',
   'featurepanel.undelete_button': 'Undelete',
   'featurepanel.feature_description_nonosm': 'Map feature __type__',
   'featurepanel.feature_description_osm': '__type__ in OpenStreetMap database',
   'featurepanel.feature_description_point': 'Map coordinates',
-  'featurepanel.show_objects_around': 'Show objects around',
-  'featurepanel.uncertain_image': 'This is the closest street view image. It may show different object.',
+  'featurepanel.show_objects_around': 'Show nearby objects',
+  'featurepanel.uncertain_image': 'This is the closest street view image, it may show other objects.',
```

Comment:
Hmm, the issue here is, that mapillary images are sometimes off-labeled, they claim different direction then reality. So unfortunately this warning was meant to emphasize this. Could you please think of another message?

Perhaps "This is the closest street view image from Mapillary, it may be wrong sometimes."

Reply:
"It may be wrong sometimes" is the part that doesn't feel very natural to me.

Maybe 
- "This is the closest street view image from Mapillary. It may show a different location"
- "This is the closest street view image from Mapilary. It may be inaccurate.

Reply:
Thanks. Both are fine. I prefer the second one a bit more. Will you change it, or should i?

Reply:
I will make the changes on my branch to avoid confusion 👍🏼